packages/excalidraw/element/elbowArrow.ts

```diff
@@ -1038,7 +991,10 @@ export const updateElbowArrowPoints = (
   // Short circuit on no-op to avoid huge performance hit
   if (
     updates.startBinding === arrow.startBinding &&
-    updates.endBinding === arrow.endBinding
+    updates.endBinding === arrow.endBinding &&
+    (updates.points ?? []).every((p, i) =>
+      pointsEqual(p, arrow.points[i] ?? Infinity),
+    )
```

Comment:
shouldn't the `points` check be `||`? But I'm not quite sure what it's doing

Reply:
`arrow.points` might have less points than the update, that is why it tries to handle the undefined case. I made a mistake for the default though, so that's fixed now.